BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/TelemetryConfigurationFactory.cs

```diff
@@ -18,11 +18,15 @@
 
     internal class TelemetryConfigurationFactory
     {
+        internal const string InstrumentationKeyEnvironmentVariable = "APPINSIGHTS_INSTRUMENTATIONKEY";
```

Comment:
aren't these defined elsewhere in the project as well?

Reply:
nope. only here.